neqo-http3/src/connection_server.rs

```diff
@@ -100,7 +102,7 @@ impl Http3ServerHandler {
     pub fn stream_close_send(&mut self, stream_id: StreamId, conn: &mut Connection) -> Res<()> {
         qdebug!([self], "Close sending side stream={}.", stream_id);
         self.base_handler.stream_close_send(conn, stream_id)?;
-        self.base_handler.stream_has_pending_data(stream_id);
+        self.base_handler.stream_might_have_pending_data(stream_id);
```

Comment:
```suggestion
```

If you look at `stream_close_send()`, you will see that this function is already called from there.  And it is already conditional.  So this can just go.